gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/WsGremlinBinaryRequestDecoder.java

```diff
@@ -76,6 +76,7 @@ protected void decode(final ChannelHandlerContext channelHandlerContext, final B
             try {
                 objects.add(serializer.deserializeRequest(messageBytes.discardReadBytes()));
             } catch (SerializationException se) {
+                logger.warn(se.getMessage());
```

Comment:
i went in a two day hole of debugging because I couldn't figure out where that a serialization problem from here was causing trouble. just wanted some additional information in the logs here to help for that sort of problem in the future. i will change the log messages to a general message and stacktrace.